jvm-packages/xgboost4j/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,9 @@
+# Set root logger level to DEBUG and its only appender to A1.
+log4j.rootLogger=DEBUG, A1
```

Comment:
When I was testing the code here, I find there is an error which is related to not find log file. DEBUG should be INFO here.